changelog.md

```diff
@@ -17,7 +17,14 @@
 3.2.2
 =========
 - Updated to Microsoft.IdentityModel.* 8.1.2
-
+- **Breaking change** (without major version change. Sorry!) 
+   ClientAssertionProviderBase.GetSignedAssertion now takes an MSAL.NET AssertionRequestOptions. See:
```

Comment:
Consider adding a link to the code:
```suggestion
   [ClientAssertionProviderBase.GetSignedAssertion](https://github.com/AzureAD/microsoft-identity-web/blob/195328cc509e3d964bb47f1877a8946b6f136f81/src/Microsoft.Identity.Web.Certificateless/ManagedIdentityClientAssertion.cs#L79) now takes an MSAL.NET AssertionRequestOptions. See:
```

Reply:
Also consider mentioning that the method name has changed from `GetSignedAssertion` to `GetClientAssertionAsync`

Reply:
good catch. Thanks @kellyyangsong